src/test/regress/expected/multi_router_planner.out

```diff
@@ -1385,6 +1385,24 @@ DEBUG:  Plan is router executable
 --------------+-------------------
 (0 rows)
 
+-- is null clause is not shard prunable so, not router plannable
```

Comment:
I do not think a detailed explanation is necessary here. router plannable-ness is already associated with pruning everything down to a single shard. will add some more info.